app/src/main/java/com/onboarding/wheatherapp/data/entity/Weather.kt

```diff
@@ -0,0 +1,5 @@
+package com.onboarding.wheatherapp.data.entity
+
+data class Weather(
+    val list: ArrayList<Data>
+)
```

Comment:
add empty line at the end of the file pls

Reply:
Solved!